src/app/(checkout)/checkout/_components/checkout-form.tsx

```
@@ -0,0 +1,121 @@
+	useEffect(() => {
+		async function fetchUserInfo() {
+			try {
+				if (!session?.user?.name) {
+					form.setValue('firstName', '')
+					form.setValue('lastName', '')
+					form.setValue('email', session?.user?.email || '')
+
+					return
+				}
+
+				const [firstName = '', lastName = ''] = session?.user?.name.split(' ')
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)